test/unit/test_classical_explainer.py

```diff
@@ -0,0 +1,46 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# Tests for model explainability SDK
```

Comment:
update comment to be a bit more specific (I think this is taken from other package?)